---
title: "Software"
---

Iâ€™ve developed several open-source tools to support genomic epidemiology and microbial genomics research. These tools aim to simplify common workflows and analyses in bacterial genomics.

For more details and to explore the code, visit my [Github profile](https://github.com/kylegontjes)

## GeneScreener

A Snakemake pipeline that automates running BLAST searches across many genomes, making gene screening scalable and reproducible. [Repository](https://github.com/kylegontjes/GeneScreener)

## ISScreener

Workflow designed to detect insertion sequences (IS elements) from Illumina whole-genome sequencing data, helping characterize mobile genetic elements. [Repository](https://github.com/kylegontjes/ISScreener)

## **CoverageStats**

Calculates coverage statistics for short-read sequencing data mapped to reference genomes, useful for quality control and comparative analyses. [Repository](https://github.com/kylegontjes/CoverageStats)

## Phylokit

A Snakemake pipeline that generates phylogenetic trees from sequence alignments, streamlining evolutionary analysis workflows. [Repository](https://github.com/Snitkin-Lab-Umich/phylokit)

## Fastools

A collection of handy scripts for manipulating FASTA files, simplifying common sequence processing tasks. [Repository](https://github.com/kylegontjes?tab=repositories)

## PhyloAMR

An R package that provides tools for phylogenetically informed analysis of antibiotic resistance and other genome-associated traits, integrating evolutionary context into genomic analyses. [Repository](https://github.com/kylegontjes/phyloAMR)
